packages/editor/src/plugins/image-gallery/components/single-image-modal.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react'
+
+import { FittingModal } from './fitting-modal'
+import type { ImageGalleryProps } from '..'
+
+interface SingleImageModalProps extends ImageGalleryProps {
+  currentImageIndex: number
+  onAddImage: () => void
+  handleMultipleImageUpload: (files: File[]) => void
+}
```

Comment:
```suggestion
interface SingleImageModalProps {
  currentImageState: ImageGalleryProps['state'][0]
  onAddImage: () => void
  handleMultipleImageUpload: (files: File[]) => void
}
```
This way, you have one prop less, and can skip line 15.

Reply:
Done